### DISCUSSION TITLE
Will expansion music (not MIDI) will be added before 1.0 ?

### DISCUSSION THREAD

**Discussion by Seth7171** [ihhub/fheroes2#5835] [General]
Hello, I have a queation
If you had the price of loyalty expansion you can play town music that were unique (not the MIDI version), will we see that before 1.0?
Thanks =)

**Comment by zenseii** [comment]
> Hello, I have a queation If you had the price of loyalty expansion you can play town music that were unique (not the MIDI version), will we see that before 1.0? Thanks =)

Hi @Seth7171 !

I'm glad you enjoy this project. Currently what you are describing is already possible with fheroes2.

You can put the Price of Loyalty music from your Heroes 2 original copy in the Music folder. Where this folder is located depends on your OS. If you need any assistance with this then don't hesitate to ask.

**Reply by Seth7171** [reply]
ohhh I see now! there is a sw and pol folders, dope! 
will you make it that we can switch the music in game without moving files in windows?

**Reply by zenseii** [reply]
> ohhh I see now! there is a sw and pol folders, dope! will you make it that we can switch the music in game without moving files in windows?

I don't think there are any plans for doing this since most players are fine using one set of tracks that they prefer. There are also many cases that would have to be handled with such an option, since there are 3 different sets of music for heroes 2 (one includes heroes 3 music) and not to mention the demo version which only has midi anyways.

Such an option would be considered much later on when we have the luxury to spend time on it, because there are much bigger issues to be solved now.

I hope that answers your question well.

**Reply by ihhub** [reply]
That's an interesting topic. GOG release indeed has two extra directories within MUSIC folder and all songs for castle's are different. The directory itself by default has PoL songs.

**Reply by Seth7171** [reply]
yes, thank you =) great work on the project I'm having a blast playing it

**Comment by Darkloke** [comment]
Thought to ask about this too before seeing this topic... Since there are 2 folders with "pol" and "sw" may be play "sw" tracks when you are playing "sw" maps and "pol" - when "pol" maps?

Also IIRC gold edition contained 4 unique town tracks - 3 of them (knight, necromancer, wizard) are (future) tracks from HoMM3 and an unique track for sorceress.
